lib/publiccloud/azure.pm

```diff
@@ -150,32 +172,18 @@ sub upload_img {
     my $tags = "openqa_created_by=$created_by";
 
     my $rg_exist = $self->resource_exist();
-
     $self->create_resources($storage_account) if (!$rg_exist);
 
     my $key = $self->get_storage_account_keys($storage_account);
 
-
+    # Note: VM images need to be a page blob type
+    assert_script_run('az storage blob upload --max-connections 4 --type page'
+          . " --account-name '$storage_account' --account-key '$key' --container-name '$container'"
+          . " --file '$file' --name '$img_name' --tags '$tags'", timeout => 60 * 60 * 2);
+    # After blob is uploaded we save the of it as its metadata.
```

Comment:
`we save the of it as its metadata` 

should be

`we save the MD5 of it as its metadata`